Automattic/jetpack
# Update/migrating videopress UI to package

#### Changes proposed in this Pull Request:

* Migrate some additional features from the plugin to the package

#### Other information:


#### Jetpack product discussion


NA

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Start with a connected site with one video uploaded to VideoPress
* Make sure only VideoPress stand-alone plugin is active

Confirm the bugs that this PR fixes before applying this branch:

* Create a new post. Add a new VideoPress block
* Click "Media Library"
* Select the videopress video
* See that there will be an error (invalid attachment ID)
* Go to the Media Library
* Click on the VideoPress video

Checkout this branch and make sure the above bugs are fixed.

* You can add a videopress video from the Media Library
* You see the player when you click on the video in the Media Library
* Also confirm that everything still works with Jetpack plugin (together with VideoPress plugin and on its own)
* Confirm that there is no video player on the left side of the modal that opens with the video details
